pkgs/development/libraries/proj/default.nix

```diff
@@ -62,10 +62,27 @@ stdenv.mkDerivation (finalAttrs: {
 
   doCheck = true;
 
-  passthru.tests = {
-    python = python3.pkgs.pyproj;
-    proj = callPackage ./tests.nix { proj = finalAttrs.finalPackage; };
-  };
+  passthru =
+    let
+      proj = finalAttrs.finalPackage;
+    in
+    {
+      withProjData = gridPackages:
+        let
+          proj-data = callPackage ./proj-data.nix { gridPackages = gridPackages; };
+        in
+        proj.overrideAttrs (final: prev: {
+          pname = prev.pname + "-with-grid-packages";
+          postInstall = ''
+            cp --recursive ${proj-data}/* $out/share/proj/
```

Comment:
Rather than copying the data across, would it be possible to instead install proj-data and *link* to it from this package? That way there's only one way to install proj-data, but proj can refer to that package.

Reply:
Unfortunately, this is not possible because `$out/share/proj/` already contains data installed by `proj`. Otherwise I would do a symlink.